helix.go

```diff
@@ -39,6 +40,10 @@ type Client struct {
 
 	baseURL      string
 	lastResponse *Response
+
+	// Logging
+	Debug  bool
```

Comment:
This field shouldn't be exported (ie. it should have a lowercase `d`).

This is actually something I want to fix in the future. There's no need to have both the _Client_ and _Options_ structs share the same fields. I'm probably going to add a _opts_ field to the _Client_ struct instead of the individual fields. Leave that to me, however, that's not a necessary for this PR.